eng/pipelines/azure-pipelines.yml

```diff
@@ -42,8 +42,8 @@ stages:
           ${{ if eq(variables._RunAsPublic, True) }}:
             vmimage: 'windows-latest'
           ${{ if eq(variables._RunAsInternal, True) }}:
-            name: NetCoreInternal-Pool
-            queue: BuildPool.Windows.10.Amd64.VS2019
+            name: NetCore1ESPool-Internal

```

Comment:
Should be `NetCore1ESPool-Svc-Internal`, I believe

Reply:
True, thanks for catching that!